tests/test_urls.py

```diff
@@ -529,7 +529,7 @@ def test_sphinx_resource_urls(app, status, warning):
             '<script src="/en/latest/_static/doctools.js"></script>',
         ]
 
-    if sphinx.version_info >= (1, 8):
+    if (1, 8) <= sphinx.version_info < (3, 4, 0):
```

Comment:
This could use a comment. Is this file not needed in Sphinx 3.4+?

Reply:
No. I added the comment in the commit message:

> This file was added to all the HTML pages. However, it was only required for
> Search page and it's not added anymore to other pages.
> See sphinx-doc/sphinx@v3.4.0/CHANGES#L87 (https://github.com/sphinx-doc/sphinx/blob/v3.4.0/CHANGES#L87)

It's basically a bugfix in Sphinx that was adding this file to all the HTML files even if it was not required.

I think that Sphinx now supports to add CSS/JS files only to specific HTML pages instead all of them.